notebooks/slsim_single_lens_simulation.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>for the example, this is too complicated. The notebook should be able to run locally. Perhaps just make some simple ones or use the lenstronomy SimulationAPI class to get you the PSF. Also this notebook does not need to be too LSST specific</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>